<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIToolBar</class>
 <widget class="QWidget" name="UIToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="HFlatPushButton" name="button_settings">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/res/icon/settings.png</normaloff>:/res/icon/settings.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HFlatPushButton" name="button_refresh">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/res/icon/reload.png</normaloff>:/res/icon/reload.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ターゲット</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Fallout 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fallout NV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Skyrim SE</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>プロファイル</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HComboBox" name="comboBox_2"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HFlatPushButton" name="pushButton_2">
     <property name="text">
      <string>翻訳</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="HFlatPushButton" name="button_info">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/res/icon/info2.png</normaloff>:/res/icon/info2.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HFlatPushButton</class>
   <extends>QPushButton</extends>
   <header>HFlatPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>HComboBox</class>
   <extends>QComboBox</extends>
   <header>HComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
